ddl/ddl_api.go

```diff
@@ -4390,7 +4390,9 @@ func (d *ddl) CreateIndex(ctx sessionctx.Context, ti ast.Ident, keyType ast.Inde
 	}
 
 	if indexInfo := t.Meta().FindIndexByName(indexName.L); indexInfo != nil {
-		err = ErrDupKeyName.GenWithStack("index already exist %s", indexName)
+		// NOTE: explicit error message. See issue #18363.
+		err = ErrDupKeyName.GenWithStack("index already exist %s; "+
+			"maybe a background job is trying to add the same index", indexName)
```

Comment:
```suggestion
			"maybe a background job is trying to add the same index, please try using `ADMIN SHOW DDL JOBS` to make a check.", indexName)
```
How about this?

Reply:
"to make a check" is too oral.
"please check by executing `ADMIN SHOW DDL JOBS`", this would be enough.

Reply:
If it's clear that another job is adding the index, "maybe" can be removed.

Reply:
done

thx a lot.